data-tool/flows/tombstone/tombstone_mappings.py

```diff
@@ -0,0 +1,87 @@
+from enum import Enum
+
+class EventFilings(str, Enum):
+    # IA
+    FILE_ICORP = 'FILE_ICORP'
+    FILE_ICORU = 'FILE_ICORU'
+    FILE_ICORC = 'FILE_ICORC'
+    CONVICORP_NULL = 'CONVICORP_NULL'
+
+    # AR
+    FILE_ANNBC = 'FILE_ANNBC'
+
+    # CoA
+    FILE_APTRA = 'FILE_APTRA'
+    FILE_NOERA = 'FILE_NOERA'  # never happened so far
+    FILE_NOCAD = 'FILE_NOCAD'
+
+    # CoD
+    FILE_NOCDR = 'FILE_NOCDR'
+
+    # # Alteration
+    # FILE_NOALA = 'FILE_NOALA'
+    # FILE_NOALB = 'FILE_NOALB'
+    # FILE_NOALU = 'FILE_NOALU'
+    # FILE_NOALC = 'FILE_NOALC'
+    # FILE_AM_PF = 'FILE_AM_PF'
+    # FILE_AM_PO = 'FILE_AM_PO'
+    # FILE_AM_TR = 'FILE_AM_TR'
+    # # FILE_AM_AR = 'FILE_AM_AR' # TODO
+    # FILE_AM_BC = 'FILE_AM_BC'
+    # FILE_AM_DI = 'FILE_AM_DI'
+    # FILE_AM_DO = 'FILE_AM_DO'
+    # FILE_AM_LI = 'FILE_AM_LI'
+    # # FILE_AM_RM = 'FILE_AM_RM' # for liquidation
+    # FILE_AM_RR = 'FILE_AM_RR'
+    # FILE_AM_SS = 'FILE_AM_SS'
+
+    # transition
+    FILE_TRANS = 'FILE_TRANS'
+
+
+    @classmethod
+    def has_value(cls, value):
+        return value in cls._value2member_map_
+    
+
+EVENT_FILING_LEAR_TARGET_MAPPING = {
```

Comment:
Planned to use `from legal_api.core import Filing as FilingCore` and leverage code in `corps/event_filing_service.py` at the first beginning, but dependency issues related to sqlalchemy kept comming up even I referred to the solution in `flows/migrate_corps_flow.py`...So eventually choose to get rid of any legal-api and sqlalchemy dependencies.